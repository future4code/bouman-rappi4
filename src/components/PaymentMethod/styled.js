import styled from 'styled-components';

export const Container = styled.div `
    width: 100%;
    height: 40px;
    display: flex;
    flex-direction: column;
    justify-content: flex-start;
    align-items: flex-start;
    padding: 15px;
`

export const StyledForm = styled.form `
    display: flex;
    flex-direction: column;
    justify-content: start;
    align-items: flex-start;
`