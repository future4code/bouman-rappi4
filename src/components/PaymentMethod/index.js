import React from 'react';
import {Container, StyledForm} from './styled'

function PaymentMethod(props) {
    return (
        <Container>
            <StyledForm>
                <span>Débito<input name="débito" onClick={props.onClick} value="Débito" type="radio"/></span>    
                <span>Crédito<input name="cartão de crédito" onClick={props.onClick} value="Crédito" type="radio"/></span> 
            </StyledForm>
        </Container>
    )
}

export default PaymentMethod;