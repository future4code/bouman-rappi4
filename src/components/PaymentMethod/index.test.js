import React from 'react'
import renderer from "react-test-renderer";
import  PaymentMethod  from './index';

describe ('teste componente PaymentMethod', () => {
    it('renders correctly', () => {
        const tree = renderer
            .create(<PaymentMethod></PaymentMethod>)
            .toJSON();
        expect(tree).toMatchSnapshot();
    })
})