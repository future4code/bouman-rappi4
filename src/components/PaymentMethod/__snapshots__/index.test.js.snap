// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`teste componente PaymentMethod renders correctly 1`] = `
<div
  className="sc-bdVaJa fSVYCz"
>
  <form
    className="sc-bwzfXH jYZWGa"
  >
    <span>
      Débito
      <input
        name="débito"
        type="radio"
        value="Débito"
      />
    </span>
    <span>
      Crédito
      <input
        name="cartão de crédito"
        type="radio"
        value="Crédito"
      />
    </span>
  </form>
</div>
`;
